cli/crates/cli/tests/scalars.rs

```diff
@@ -1,35 +1,197 @@
 mod utils;
 
 use serde_json::{json, Value};
-use utils::consts::{SCALARS_MUTATION, SCALARS_QUERY, SCALARS_SCHEMA};
+use utils::client::Client;
+use utils::consts::{SCALARS_CREATE_OPTIONAL, SCALARS_CREATE_REQUIRED, SCALARS_SCHEMA};
 use utils::environment::Environment;
 
+trait ClientScalarExt {
```

Comment:
Is this used anywhere else? A bit confused as to why it's a trait

Reply:
I'm just adding some methods to the `client`. For this, I need a trait & impl. But I agree that just using functions shouldn't hurt readability, I guess I'm just too used to this pattern from Kotlin. I don't mind changing if you prefer functions in this case.

Reply:
I see, it's fine, I thought you were using a trait for re-use which didn't really happen here

Reply:
But to be honest I'm not a big fan of the API I came up with for this. I'll probably try to change it. `transform` and `error_msg` are just poor method names.